nvflare/fuel/f3/streaming/tools/receiver.py

```diff
@@ -40,25 +40,30 @@ def blob_cb(self, stream_future: StreamFuture, *args, **kwargs):
 
 
 if __name__ == "__main__":
-    setup_log(logging.DEBUG)
+    setup_log(logging.INFO)
     url = "tcp://localhost:1234"
     receiver = Receiver(url)
     time.sleep(2)
     result = None
+    last = 0
     while True:
         if receiver.get_futures:
             for sid, fut in receiver.get_futures().items():
                 if fut.done():
                     result = fut.result()
                     break
                 else:
-                    print(f"{sid} Progress: {fut.get_progress()}")
+                    progress = fut.get_progress()
+                    print(f"{sid} Progress: {progress} Delta:{progress - last}")
```

Comment:
change to use logger?

Reply:
This is output of a CLI test tool. print is better.